h2/events.py

```diff
@@ -356,22 +356,50 @@ def __repr__(self):
         )
 
 
-class PingAcknowledged(Event):
+class PingReceived(Event):
     """
-    The PingAcknowledged event is fired whenever a user-emitted PING is
-    acknowledged. This contains the data in the ACK'ed PING, allowing the
+    The PingReceived event is fired whenever a PING is received. It contains
+    the 'opaque data' of the PING frame.
```

Comment:
Can we add a note here that this event requires no action on the part of the user, as hyper-h2 automatically replies to PING frames?